File: benchmark/results/python-invocation-script/run-001/a-mcp/output/billing/services/invoice_service.py
from typing import List, Optional
from billing.aggregates.invoice import Invoice

"""InvoiceService service."""
class InvoiceService:
    def create(self, input: Invoice) -> Invoice:
        raise NotImplementedError('not implemented')
    def findById(self, id: str) -> Optional[Invoice]:
        raise NotImplementedError('not implemented')
    def list(self, limit: int) -> List[Invoice]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

File: benchmark/results/java-invocation-inline/run-005/b-baseline/output/com/metaengine/demo/billing/services/InvoiceService.java
package com.metaengine.demo.billing.services;

import com.metaengine.demo.billing.aggregates.Invoice;
import java.util.List;

/** Application service for the Invoice aggregate. */
public class InvoiceService {

    /** Create a new Invoice from a partial input. */
    public Invoice create(Invoice input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an Invoice by id; returns null if not found. */
    public Invoice findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List Invoices, capped at the supplied limit. */
    public List<Invoice> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete an Invoice by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}
